src/hooks/useWalletSectionsData.ts

```diff
@@ -60,26 +59,14 @@ export default function useWalletSectionsData({
   const hiddenAssets = useUserAssetsStore(state => state.hiddenAssets);
   const isLoadingUserAssets = useUserAssetsStore(state => state.getStatus('isInitialLoad'));
   const sortedAssets = useUserAssetsStore(state => state.legacyUserAssets);
-  const positionsData = usePositionsStore(state =>
-    state.getData({
-      address: accountAddress,
-      currency: nativeCurrency,
-      chainIds: useBackendNetworksStore.getState().getSupportedPositionsChainIds(),
-    })
```

Comment:
Both stores are already listening to upstream param stores in the `params` setter

```ts
params: {
      address: $ => $(userAssetsStoreManager).address,
      currency: $ => $(userAssetsStoreManager).currency,
      chainIds: $ => $(useBackendNetworksStore, s => s.getSupportedPositionsChainIds()),
    },
```